Shrine/app/src/main/java/com/authentication/shrine/ui/viewmodel/RegistrationViewModel.kt

```diff
@@ -79,7 +79,7 @@ class RegistrationViewModel @Inject constructor(
                             )
                         }
                     }
-                } catch(e: Exception) {
+                } catch (e: Exception) {
```

Comment:
Might be helpful to do the reformat first and land that CL. Lot of comma and space only changes in the CL.

#tioli